### DISCUSSION TITLE
Import json to mdx file

### DISCUSSION THREAD

**Discussion by tienlx97** [mdx-js/mdx#2207] [Q&A] [answered]
I use nextjs + mdx to write blog. This is my folder contain all mdx file:
content
&nbsp;&nbsp;&nbsp;blog
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;hello-mdx <-- folder
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;index.md
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;data.json

In **index.md** I have 2 components `<Comp1 />` and `<Comp2 />`: 

``` html
<Comp1>
   <Item key="pikachu" type="light" />
   <Item key="raichu" type="light" />
   <Item key="chikorita" type="leaf" />
   <Item key="mews" type="physic" />
</Comp1>

<Comp2>
   <Item2 append="1" key="pikachu" type="light" />
   <Item2 append="2" key="raichu" type="light" />
   <Item2 append="3" key="chikorita" type="leaf" />
   <Item2 append="4" key="mews" type="physic" />
</Comp2>
```
Because data in `<Comp1 />` and `<Comp2 />` are *static*. So I create a json file name **data.json**

``` json
{
 "pikachu": "light",
 "raichu": "light",
 "chikorita": "leaf"
}
```

Can I import **data.json** to **index.md** ? In **index.md** I should change 
``` html
<Comp1 jsonName="data" /> 
<Comp2 jsonName="data" />
```

**Comment by ChristianMurphy** [comment, marked as answer]
This looks like a duplicate of https://github.com/orgs/mdx-js/discussions/2206 ?

And the answer is the same:

> Likely not though: you can't import json yet in ESM. In the future you could use import assertions, but they're not supported in most places yet.
